plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -320,6 +320,108 @@ fu_thunderbolt_device_get_attr_uint16(FuThunderboltDevice *self, const gchar *na
 	return (guint16)val;
 }
 
+static gboolean
+fu_thunderbolt_device_set_port_offline(FuUdevDevice *device, GError **err)
+{
+	if (!fu_udev_device_write_sysfs(device, "usb4_port1/offline", "1", err)) {
```

Comment:
I think you might need an extra check that this file exists first.  Think about the case that this file isn't existing, this function should return true immediately right?

Reply:
as long as the usb4 port exists and kernel driver is loaded this file should exist, I can do a pre check if needed but I cant find any case where this file will be missing and we trigger retimer nvm update.

Reply:
fixed in https://github.com/fwupd/fwupd/commit/98336139be858dd20f4089bfea421f0fda9031b1